gg_analyze/log.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import argparse
+import math
+import msgpack
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interp
+import scipy.signal as signal
+import seaborn as sns
+import serial
+import sys
+
+parser = argparse.ArgumentParser(description='Log gufu_gud data.')
+parser.add_argument('device', help='Device name', required=True)
```

Comment:
`required` is incompatible with positional arguments, positional arguments are always required by default.